index.html

```diff
@@ -10,10 +10,76 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
-
+<div class="container">
```

Comment:
```suggestion
```
The container styles don't seem to do anything differently, so we can delete this div container